simulation/agents.py

```diff
@@ -444,9 +444,13 @@ def choose_heating_system(
     ):
 
         weights = []
+        multiple_cap = 50  # An arbritrary cap to prevent math.exp overflowing
```

Comment:
It seems like the maximum we could use here is `700`. Let me know if you want to change it, or leave it as-is